products/compute/api.yaml

```diff
@@ -4856,6 +4856,69 @@ objects:
         description: |
           The default port used if the port number is not specified in the
           network endpoint.
+  - !ruby/object:Api::Resource
+    name: 'NodeGroup'
+    kind: 'compute#NodeGroup'
+    base_url: projects/{{project}}/zones/{{zone}}/nodeGroups
+    create_url: projects/{{project}}/zones/{{zone}}/nodeGroups?initialNodeCount={{size}}
+    has_self_link: true
+    description: |
+      Represents a NodeGroup resource to manage a group of sole-tenant nodes.
+    input: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Sole-Tenant Nodes': 'https://cloud.google.com/compute/docs/nodes/'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/v1/nodeGroups'
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/zones/{{zone}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::Async::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::Async::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'zone'
+        resource: 'Zone'
+        imports: 'selfLink'
+        required: true
+        description: |
+          Zone where this node group is located
+    properties:
+      - !ruby/object:Api::Type::Time
+        name: 'creationTimestamp'
+        description: 'Creation timestamp in RFC3339 text format.'
```

Comment:
Nit: I'd be inclined to use the `description: |` form every time for consistency.